wormhole-connect/src/config/tokens.ts

```diff
@@ -21,6 +21,8 @@ import { getTokenMetadataFromRpc } from 'utils/tokens';
 
 const TOKEN_CACHE_VERSION = 1;
 
+const HAS_LOCALSTORAGE = typeof localStorage !== 'undefined';
```

Comment:
this was throwing errors in server side rendering because `localStorage` is not defined